### PR TITLE
[4.17] merge efs to aws byo iam and s3 cloudfront to byo route53

### PR DESCRIPTION
**Pull request by jianlinliu**
To save aws cost, merge efs to aws byo iam and s3 cloudfront to byo route53 profile

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @yunjiang29 @Phaow to review

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml [hunk 1]
```diff
@@ -93,18 +97,21 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-profile
-- as: aws-ipi-byo-iam-role-arm-f60
+- as: aws-ipi-byo-iam-role-efs-arm-f60-custom-cert
```
Comment (Phaow) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml:100]: @jianlinliu Hmm, needs a bit info, is this `-f60` frequency not as a suffix expected? The same in https://github.com/openshift/release/pull/56160/files#diff-fd088e7048faa35212d70b16f01645b6f15276135dabfd8af796458bf88a4179R65 . It is not consistent with 4.15 PR ->
https://github.com/openshift/release/pull/56126/files#diff-6c9d8f80cbd2256599a522458fa761a0aa949af8c99c8b7139b6456005d0ee11R67 .
Seems should be ->
```suggestion
- as: aws-ipi-byo-iam-role-efs-arm-custom-cert-f60
```
right?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml [hunk 2]
```diff
@@ -134,32 +134,38 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-profile
-- as: aws-ipi-byo-iam-role-arm-f2
+- as: aws-ipi-byo-iam-role-efs-arm-f2-custom-cert
```
Comment (Phaow) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml:137]: The same above.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml (hunk 1 above)
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml:100]: `custom-cert` is added as a test chain, for test chain, we usually append it after frequency field, the same example - `destructive`

Comment (Phaow) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml:100]: Got it, thanks for the double confirm!

**Pull request merged**